### ISSUE TITLE
Fix reactive on pm admin/list

### ISSUE THREAD

**Issue by JeffAbrahamson** [transport-nantes/tn_web#904]
The reactive fails pretty hard.  Here's a 960 px window.

![image](https://user-images.githubusercontent.com/1220683/186597866-e72fd18b-f042-4f13-8258-072b583cab02.png)

**Comment by Shriukan33** [comment]
I think tabular data are first narrowed to try to fit until a minimum, and then you scroll horizontally.
I could  make the minimum size of columns bigger, but I don't think there is a way to present that many columns in a single small screen.

**Comment by JeffAbrahamson** [comment]
> I think tabular data are first narrowed to try to fit until a minimum, and then you scroll horizontally. I could make the minimum size of columns bigger, but I don't think there is a way to present that many columns in a single small screen.

There's a CSS / bootstrap thingy that lets you say "above this size, I want to layout like this; below this size, I want to layout this other way".

**Comment by Shriukan33** [comment]
> > I think tabular data are first narrowed to try to fit until a minimum, and then you scroll horizontally. I could make the minimum size of columns bigger, but I don't think there is a way to present that many columns in a single small screen.
> 
> There's a CSS / bootstrap thingy that lets you say "above this size, I want to layout like this; below this size, I want to layout this other way".

Yes, usually things that are side by side on larger screen go in column, thanks to flexboxes. 
In tables though, you can't put a column under another, it would make the table unreadable. 

A possibility would be to display some columns and hide others on smaller screens. I did something similar in a OC project where on mobile you get 3 columns and 5 on bigger screens. (the two hidden were less important)
